exercises/meetup/meetup_test.rb

```diff
@@ -550,4 +548,50 @@ def test_last_sunday_of_april_2013
     assert_equal Date.new(2013, 4, 28),
       Meetup.new(4, 2013).day(:sunday, :last)
   end
+
+  def test_last_wednesday_of_february_2012
+    skip
+    assert_equal Date.new(2012, 2, 29),
+      Meetup.new(2, 2012).day(:wednesday, :last)
```

Comment:
What about extracting an `expected` variable so the assert fits on one line?

Reply:
It might not even be the one line that bugs me, I think it's the amount of information I have to care about.

I'd probably split this into 3 lines.

```
expected_date = Date.new(2012, 2, 29)
subject =  Meetup.new(2, 2012)
assert_equal expected_date, subject.day(:wednesday, :last)
```

On writing this example I've discovered that it's probably also the interface of Meetup that bugs me. (Again, not a problem with this PR specifically)